packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -67,6 +68,10 @@ import 'runtime.dart';
 ///  * The [Scaffold]'s floating action button position and animation features
 ///    are not supported.
 ///
+///  * [DropdownButton] takes a list of objects representing the
+///    [DropdownMenuItem]s. Each object may contain `onTap`, `value`, `enabled`
+///   and `child`. The `child` param is required.
```

Comment:
nit: indentation

Reply:
nit: avoid abbreviations ("param") (see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations)